modules/cnf-du-annotating-workloads.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-annotating-workloads_{context}"]
+
+= Annotating workloads
+
+Two annotations must be available to isolate a workload into partitioned resources. Annotate the containing namespace with `workload.openshift.io/allowed` and the pod
+with `target.workload.openshift.io`.
+
+Required OCP platform pods and namespaces are already annotated. To add extra pods to the partition, such as supported day 2 operators or any user-added catalog sources, annotate both the namespace and pod on creation.
+
+The namespace annotation allows cluster administrators to control which workloads have access to the partitioned resources. Only pods in namespaces containing the annotation `workload.openshift.io/allowed: {<workload-type> [, <workload-type]}` will be considered. Annotated pods that are contained in an unannotated Namespace will fail to admit to the cluster. In this case, an error is raised indicating the pod is forbidden because the namespace does not allow the `workload type <workload-type>`.
```

Comment:
I think "partitioned resources" muddies the concepts for readers.  I believe a viable alternative is:

After you add the namespace annotation, you can control which workloads have access to the reserved CPU resources.

I believe "reserved" is the concept to emphasize here.  Especially if it is related to this entry about [specifying CPUs for host host processes](https://docs.openshift.com/container-platform/4.7/release_notes/ocp-4-7-release-notes.html#ocp-4-7-nodes-crio-support-cpus-for-node-process) from the 4.7 release notes.  The language there matches my expectations.

nit: I don't think you want the braces instead of the \< and \>.

nit: you can use lowercase "namespace"

nit: The "Annotated pods that are..." sentence--can it be revised to use active voice?  "fail to admit" is new to me.  Previously, I've read about "scheduling" in case that term is the same idea that is touched on in this example for [taints and tolerations](https://docs.openshift.com/container-platform/4.7/post_installation_configuration/node-tasks.html#nodes-scheduler-taints-tolerations-about_post-install-node-tasks).  Maybe it's not, otherwise taints and tolerations could have been used.